### PR TITLE
getMaxSpeechInputLength() in flutter_tts

### PR DESCRIPTION
**Pull request by divshekhar**
Issue reference: `getMaxSpeechInputLength()` in flutter_tts #245

### CODE REVIEW

**Review by dlutton — CHANGES_REQUESTED** [review]
Please add changes for iOS or remove the comment about it being supported on Android and iOS.

File: lib/flutter_tts.dart [hunk 1]
```diff
@@ -121,6 +121,12 @@ class FlutterTts {
   /// ***iOS supported only***
   Future<dynamic> pause() async => _channel.invokeMethod('pause');
 
+  /// [Future] which invokes the platform specific method for getMaxSpeechInputLength
+  /// ***Android and iOS supported only***
```
[the remark this answers is not in the document]
Comment (divshekhar) [reply lib/flutter_tts.dart:125]: Ohh sorry for inconvenience, I am going to remove the comment for iOS.

File: example/lib/main.dart [hunk 2]
```diff
@@ -193,7 +194,8 @@ class _MyAppState extends State<MyApp> {
                   _btnSection(),
                   _engineSection(),
                   _futureBuilder(),
-                  _buildSliders()
+                  _buildSliders(),
+                  _getMaxSpeechInputLengthSection(),
```
Comment (dlutton) [inline example/lib/main.dart:198]: since this is only supported on android, you should check for that.  `if (isAndroid) `

**Review by dlutton — CHANGES_REQUESTED** [review]

File: example/lib/main.dart (hunk 2 above)
Comment (divshekhar) [reply example/lib/main.dart]: Done, please let me know if I missed something!

**Pull request merged**